dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/IconOperationParams.java

```diff
@@ -49,6 +51,8 @@ public class IconOperationParams {
   private Date createdEndDate;
   private Date lastUpdatedStartDate;
   private Date lastUpdatedEndDate;
+  private boolean paging = true;
+  private Pager pager = PagerUtils.DEFAULT_PAGING;
```

Comment:
I would not share this default pager as its API allows for mutations. I don't think we can trust it in that aspect.